### PR TITLE
tree-wide: rm {lua,python3} bindings, rm deprecated binaries, rm deprecated templates, add --{disable,enable}-{commands,tools} flag

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
jenkins: test this please

**Comment by brauner** [comment]
@stgraber, good to go!

File: src/tests/lxc-test-apparmor-mount [hunk 1]
```diff
@@ -158,6 +158,9 @@ if [ -f /etc/lsb-release ]; then
 fi
 
 run_cmd lxc-create -t download -n $cname -- -d ubuntu -r $release -a $ARCH
+echo "lxc.include =" >> $HDIR/.local/share/lxc/$cname/config
+echo "lxc.include = /usr/share/lxc/config/common.conf" >> $HDIR/.local/share/lxc/$cname/config
+echo "lxc.include = /usr/share/lxc/config/userns.conf" >> $HDIR/.local/share/lxc/$cname/config
```
Comment (stgraber) [inline src/tests/lxc-test-apparmor-mount]: We should drop that particular commit, we'll have to make sure our images don't have that issue before we merge it.

**Comment by brauner** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**